base/interactiveutil.jl

```diff
@@ -580,22 +580,22 @@ end
 
 # `methodswith` -- shows a list of methods using the type given
 """
-    methodswith(typ[, module or function][, showparents::Bool=false])
+    methodswith(typ[, module or function]; parents::Bool=false])
```

Comment:
I think this should be called `supertypes`, following e.g. the `supertype` function.